docs/specs/markdown.yml

```diff
@@ -858,15 +860,32 @@ outputs:
       "no-loc": ["method", "SDK"]
     }
 ---
-# remove domain for alternative hostname for absolute links
+# remove domain for alternative hostname for absolute links when removeHostName is true
 inputs:
   docfx.yml: |
+    hostName: docs.microsoft.com
     alternativeHostname: new-docs.microsoft.com
     removeHostName: true
   docs/a.md: |
-    [A link](https://new-docs.microsoft.com/en-us/azure)
+    [A link](https://docs.microsoft.com/en-us/azure#bookmark)
+    [Another link](https://new-docs.microsoft.com/zh-cn/azure?view=moniker-1)
 outputs:
   docs/a.json: |
     {
-       "conceptual": "<p><a href=\"/en-us/azure\" data-linktype=\"absolute-path\">A link</a></p>",
+       "conceptual": "<p><a href=\"/en-us/azure#bookmark\" data-linktype=\"absolute-path\">A link</a><a href=\"/zh-cn/azure?view=moniker-1\" data-linktype=\"absolute-path\">Another link</a></p>",
+    }
+---
+# keep domain for alternative hostname for absolute links when removeHostName is false
+inputs:
+  docfx.yml: |
+    hostName: docs.microsoft.com
+    alternativeHostname: new-docs.microsoft.com
+    removeHostName: false
+  docs/a.md: |
+    [A link](https://docs.microsoft.com/en-us/azure)
+    [Another link](https://new-docs.microsoft.com/zh-cn/azure?view=moniker-1)
+outputs:
+  docs/a.json: |
+    {
+       "conceptual": "<p><a href=\"https://docs.microsoft.com/en-us/azure#bookmark\" data-linktype=\"external\">A link</a><a href=\"https://docs.microsoft.com/zh-cn/azure?view=moniker-1\" data-linktype=\"external\">A link</a></p>",
```

Comment:
Why change the new new domain to the old one

Reply:
Please review the lastast commit. This has been changed.